css/icons.scss

```diff
@@ -46,11 +46,14 @@
 	&.in-call {
 		.app-navigation-toggle,
 		.action-item__menutoggle {
-			color: #fff;
+			color: #ffffff;
+			filter: drop-shadow(1px 1px 4px var(--color-box-shadow));
 		}
 	}
 
 	.forced-white {
+		filter: drop-shadow(1px 1px 4px var(--color-box-shadow));
```

Comment:
We don’t need both the circle _and_ the drop-shadow, right? We can remove the drop-shadows I’d say

Reply:
This also affects the other icons which don't have a circle around them.
But we can have a look at it again when this is live

Reply:
Yeah, no big deal :) Looks nicer if the icons with the circle don’t have the drop-shadow, just something for the long list cc @ma12-co